muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/records.tsx

```diff
@@ -111,47 +107,15 @@ function getActivity(props: RecordsProps, workspace: WorkspaceType){
     }
     let evaluablesCompleted = workspace.studentActivity.evaluablesPassed + workspace.studentActivity.evaluablesSubmitted +
       workspace.studentActivity.evaluablesFailed + workspace.studentActivity.evaluablesIncomplete;
-    return <div className="workspace-activity workspace-activity--studies">
-    
-      {workspace.studentActivity.evaluablesTotal ? <ProgressBarLine containerClassName="workspace-activity__progressbar workspace-activity__progressbar--studies" initialAnimate options={{
-        strokeWidth: 1,
-        duration: 1000,
-        color: "#ce01bd",
-        trailColor: "#f5f5f5",
-        trailWidth: 1,
-        svgStyle: {width: "100%", height: "4px"},
-        text: {
-          className: "workspace-activity__progressbar-label",
-          style: {
-            left: workspace.studentActivity.evaluablesDonePercent === 0 ? "0%" : null,
-            right: workspace.studentActivity.evaluablesDonePercent === 0 ? null : 100 - workspace.studentActivity.evaluablesDonePercent +  "%"
-          }
-        }
-      }}
-      strokeWidth={1} easing="easeInOut" duration={1000} color="#ce01bd" trailColor="#f5f5f5"
-      trailWidth={1} svgStyle={{width: "100%", height: "4px"}}
-      text={evaluablesCompleted + "/" + workspace.studentActivity.evaluablesTotal}
-      progress={workspace.studentActivity.evaluablesDonePercent/100}/> : null}
-    
-      {workspace.studentActivity.exercisesTotal ? <ProgressBarLine containerClassName="workspace-activity__progressbar workspace-activity__progressbar--studies" initialAnimate options={{
-        strokeWidth: 1,
-        duration: 1000,
-        color: "#ff9900",
-        trailColor: "#f5f5f5",
-        trailWidth: 1,
-        svgStyle: {width: "100%", height: "4px"},
-        text: {
-          className: "workspace-activity__progressbar-label",
-          style: {
-            left: workspace.studentActivity.exercisesDonePercent === 0 ? "0%" : null,
-            right: workspace.studentActivity.exercisesDonePercent === 0 ? null : 100 - workspace.studentActivity.exercisesDonePercent + "%"
-          }
-        }
-      }}
-      strokeWidth={1} easing="easeInOut" duration={1000} color="#ff9900" trailColor="#f5f5f5"
-      trailWidth={1} svgStyle={{width: "100%", height: "4px"}}
-      text={workspace.studentActivity.exercisesAnswered + "/" + workspace.studentActivity.exercisesTotal}
-      progress={workspace.studentActivity.exercisesDonePercent/100}/> : null}
+    return <div className="application-list__header-secondary">
```

Comment:
Can this be header-secondary as it will be placed after header-primary with other "secondary" elements like evaluation request and assessment elements?

In the end we end up in the following markup with multiple secondaries:

`<div class="application-list__header">`
`<div class="application-list__header-primary">...</div>`
`<div class="application-list__header-secondary">...</div>`
`<div class="application-list__header-secondary">...</div>`
`<div class="application-list__header-secondary">...</div>`
`</div>`

Is this desirable to have it this way? Maybe we should place all those elements inside of header-secondary elements and name them individually?

This has been the case earlier before this activity badge as well so not really needed to be changed here now.

Reply:
Yeah, I noticed it too. The secondary should ideally contain all of the badges.

Reply:
Well, it's fixed here and now